torchreid/integration/nncf/compression.py

```diff
@@ -165,6 +165,18 @@ def get_default_nncf_compression_config(h, w):
     return nncf_config_data
 
 
+def register_custom_modules():
+    # Register custom modules.
+    # Users of nncf should manually check every custom
+    # layer with weights which should be compressed and
+    # in case such layers are not wrapping by nncf,
+    # wrap such custom module by yourself.
+    from nncf.torch import register_module
+    from timm.models.layers.conv2d_same import Conv2dSame
+    wrapper = register_module(ignored_algorithms=[])
+    wrapper(Conv2dSame)
```

Comment:
There was a suggestion from @vshampor 

```suggestion
    register_module(ignored_algorithms=[])(Conv2dSame)
```